ci-operator/step-registry/openshift-extended/isc-test/web-tests/openshift-extended-isc-test-web-tests-commands.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+
+if [ -f "${SHARED_DIR}/proxy-conf.sh" ] ; then
+  source "${SHARED_DIR}/proxy-conf.sh"
+fi
+
+pwd && ls -ltr
+cd frontend || exit 0
+pwd && ls -ltr
+
+## skip all tests when console is not installed
+if ! (oc get clusteroperator console --kubeconfig=${KUBECONFIG}) ; then
+  echo "console is not installed, skipping all console tests."
+  exit 0
```

Comment:
Usually, in shell, return code 0: This means the command executed successfully without errors. It indicates "success." Better to use non-zero here. other return code also follows this rule.

Reply:
I don't want to mark the whole test job as failed if this test failed.

Reply:
Okay, see.